### ISSUE TITLE
Anvil item naming seems to be extremely buggy.

### ISSUE THREAD

**Issue by mibby** [PaperMC/Paper#552]
PaperSpigot dev 989
No plugins.

Naming items in an anvil seems to be quite buggy. Sometimes it works, most of the time it does not. 

* In survival gamemode.
* Have 6 experience levels.
* Have 3 pumpkin pie.
* Put all 3 pumpkin pie in an anvil.
* Change the name to 'test'
* It states it costs only 1 experience level.
* Trying to click on the final named item output, it snaps back to the input column, unable to be named.

If you take 1 pumpkin pie at a time, you are able to name the item. This is odd because I was able to name a stack of 64 stone at once with it only taking 1 level when testing shortly after. But when trying to reproduce again, I'm now not able to name 64 stone. Only 1 at a time. 

Did the minecraft update break something?

**Comment by MatrixTunnel** [comment]
This might be related to how Mojang patched combining Infinity and Mending enchants on the same bow

**Comment by aikar** [comment]
Are you on a non english client language?

**Comment by aikar** [comment]
And does this occur in CraftBukkit, Spigot, Vanilla?

**Comment by mikroskeem** [comment]
In Vanilla I'm not able to rename pumpkin pies if I insert more than one of them. Same thing happens in Spigot and CB.  
In Paper I'm able to insert more than one, but I have to take pies out one by one and type desired name every time again, doesn't matter if [output box shows pie with name I set before](https://0x0.st/pwU.png).

Tested with English client language (using English nevertheless, native language translations are extremely retarded imo)

No plugins were installed.

CB version: `CraftBukkit version git-Bukkit-613be0d (MC: 1.11.2) (Implementing API version 1.11.2-R0.1-SNAPSHOT)`  
Spigot version: `CraftBukkit version git-Spigot-7d78b81-613be0d (MC: 1.11.2) (Implementing API version 1.11.2-R0.1-SNAPSHOT)`  
Paper version: `Paper version git-Paper-"7a1b3937" (MC: 1.11.2) (Implementing API version 1.11.2-R0.1-SNAPSHOT)`
